letsencrypt/display/ops.py

```diff
@@ -114,23 +114,36 @@ def pick_configurator(
         config, default, plugins, question,
         (interfaces.IAuthenticator, interfaces.IInstaller))
 
-
-def get_email():
+def get_email(more=False, invalid=False):
     """Prompt for valid email address.
 
+    :param bool more: -- explain why the email is strongly advisable, but how to
```

Comment:
You should remove the `--`. A hyphen is added for every parameter when the docs are formatted so this looks odd.